apps/cyberstorm-storybook/stories/components/Switch.stories.tsx

```diff
@@ -1,42 +1,27 @@
 import { StoryFn, Meta } from "@storybook/react";
 import { Switch } from "@thunderstore/cyberstorm";
-import React, { CSSProperties, useState } from "react";
+import React, { useState } from "react";
 
 export default {
   title: "Cyberstorm/Components/Switch",
   component: Switch,
 } as Meta<typeof Switch>;
 
-const backgroundStyle: CSSProperties = {
-  display: "flex",
-  flexDirection: "column",
-  gap: "var(--gap--16)",
-  backgroundColor: "var(--color-purple--4)",
-  padding: "var(--gap--16)",
+const defaultArgs = {
+  disabled: false,
```

Comment:
`false` is the default value for this prop, so strictly speaking this isn't required. I guess it doesn't hurt either. It's just that not including it would make the change smaller since now this is passed to components for no reason.

Reply:
My first intention was to do it without passing values here, but couldn't find way to do this without changing it to use defaultArgs and because of that couldn't just pass empty variable either. 
More than happy to hear out what would have been correct way to do it!